platform/macos/src/MGLMapView.h

```diff
@@ -247,7 +247,8 @@ MGL_EXPORT IB_DESIGNABLE
  If the value of this property is smaller than that of the `minimumZoomLevel`
  property, the behavior is undefined.
 
- The default value of this property is 20.
+ The default value of this property is 22. The upper bound for this property
+ is 25.5.
```

Comment:
I think I originally left out the upper bound because setting the property to something higher like 30 would silently get capped to 25.5 anyways.